migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -108,7 +110,8 @@ BEGIN
         transform_mine_type_code(mms_new.mine_typ),
         mms_new.lat_dec    ,
         mms_new.lon_dec    ,
-        transform_major_mine_ind(mms_new.min_lnk)
+        transform_major_mine_ind(mms_new.min_lnk),
+        CASE WHEN lower(mms.mmsmin.mine_nm) LIKE '%delete%' OR lower(mms.mmsmin.mine_nm) LIKE '%deleted%' OR lower(mms.mmsmin.mine_nm) LIKE '%reuse%' THEN TRUE ELSE FALSE
```

Comment:
Your choice, but this seems like a good option for a function, like the `transform_` functions above it. I stuck repeated business logic into functions to help DRY things up